.golangci.next.reference.yml

```diff
@@ -1254,7 +1254,7 @@ linters-settings:
         failOn: dsl,import
         # Comma-separated list of file paths containing ruleguard rules.
         # If a path is relative, it is relative to the directory where the golangci-lint command is executed.
-        # The special '${configDir}' variable is substituted with the absolute directory containing the golangci config file.
+        # The special '${configDir}' variable is substituted with the absolute directory containing the golangci-lint config file.
```

Comment:
I propose moving all typo-related commits to a separate PR (so we can merge it quicker). This will make this PR easier to review because of fewer changes.

Reply:
Most of the typos are related to the topic of the PR, also there is one commit by changes to ease the review.

So I will not move them.